package.json

```diff
@@ -57,7 +57,6 @@
   "lint-staged": {
     "**/*.sol": [
       "npm run lint:sol",
-      "npm run lint:sol:prettier:fix",
```

Comment:
wait when should this be run then?

Reply:
when u commit, if it show linting error, run that command to fix errors.

Reply:
why not just run it every time?

Reply:
if 'npm run lint:sol' fail, that command will not run
if 'npm run lint:sol' succeed, that command will not do anything when it run because basically linting is good, right ?

So no need for that command IMO.